src/providers/RainbowKitSiweNextAuthProvider.tsx

```diff
@@ -0,0 +1,98 @@
+import {
```

Comment:
Not sure I understand this change - mind explaining what this is for?

Reply:
Because it's not possible to change nonce and issuedAt values to get deterministic signature.